class.wp-cldr.php

```diff
@@ -110,9 +110,16 @@ public function get_CLDR_data( $locale, $bucket ) {
 		$dir = __DIR__;
 		$data_file_name = "$dir/cldr/main/" . $CLDR_locale . '/' . $bucket . '.json';
 
+		// if no language-country locale CLDR file, fall back to a language-only CLDR file name
 		if ( ! file_exists( $data_file_name ) ) {
-			$data_file_name = "$dir/cldr/main/en/" . $bucket . '.json';
-			$CLDR_locale = 'en';
+			if ( substr ( $locale, 2, 1 ) == '-' || substr ( $locale, 2, 1 ) == '_' ) {
```

Comment:
The return of this `substr` call should be stored to a variable & referenced twice instead of calling the function twice w/ identical input.

Also, there should be no space between a function name and the parenthesis:
`substr (` should be `substr(`